htlcswitch/link_test.go

```diff
@@ -5630,6 +5630,14 @@ func TestCheckHtlcForward(t *testing.T) {
 		}
 	})
 
+	t.Run("insufficient fee and insufficient channel balance", func(t *testing.T) {
+		result := link.CheckHtlcForward(hash, 100005, 100000,
```

Comment:
nit: func call style

```
result := link.CheckHtlcForward(
        hash, 100005, 100000,200, 150, 0,
)
```

Reply:
nit: maybe we can update the comment so it makes more sense why the tests exists, something like
`insufficient fee error takes preference over insufficient channel balance`